packages/transforms/rename/test/__snapshots__/wrapRename.spec.ts.snap

```diff
@@ -104,7 +104,7 @@ exports[`rename should only affect specified field argument 1`] = `
 "type Query {
   my_user: MyUser!
   my_book: MyBook!
-  profile(profileId: ID!): Profile
+  profile(profileId: ID!, profile: String): Profile
```

Comment:
The above is just a side note, totally unrelated to this PR.
The test for rename transform has been modified to cover fields that have multiple arguments.
The implementation in `bare` mode works as expected; instead `wrap` mode is broken and because of this, line 532 in the test is commented out.